tests/core/test_accounts.py

```diff
@@ -362,24 +401,56 @@ def test_eth_account_recover_transaction_from_eth_test(acct, transaction):
     assert acct.recoverTransaction(raw_txn) == expected_sender
 
 
-def test_eth_account_encrypt(acct, web3js_key, web3js_password):
-    encrypted = acct.encrypt(web3js_key, web3js_password)
+@pytest.mark.parametrize(
+    'private_key, password, expected_decrypted_key',
+    [
+        (
+            web3js_key(),
+            web3js_password(),
+            to_bytes(hexstr=web3js_key()),
+        ),
+        (
+            web3js_private_key(web3js_key()),
+            web3js_password(),
+            web3js_private_key(web3js_key()),
```

Comment:
[The return type of `acct.decrypt` should always be `HexBytes`](http://eth-account.readthedocs.io/en/latest/eth_account.html#eth_account.account.Account.decrypt) (this appears to be expecting a `PrivateKey`).

Reply:
Along the same lines as the comment above:
```
>>> decrypted_key
b'L\x08\x83\xa6\x91\x02\x93}b1G\x1b]\xbbb\x04\xfeQ)ap\x82y*\xe4h\xd0\x1a?6#\x18'
>>> expected_decrypted_key
'0x4c0883a69102937d6231471b5dbb6204fe5129617082792ae468d01a3f362318'
>>> type(decrypted_key)
<class 'bytes'>
>>> type(expected_decrypted_key)
<class 'eth_keys.datatypes.PrivateKey'>
>>> keys.PrivateKey(decrypted_key)
'0x4c0883a69102937d6231471b5dbb6204fe5129617082792ae468d01a3f362318'
>>> decrypted_key == expected_decrypted_key
True
>>> type(decrypted_key) == type(expected_decrypted_key)
False
```

Reply:
Fixed here: https://github.com/ethereum/eth-account/pull/30/commits/b501da3a3ba80fb9e62ce58b20575ca91f2548e7#diff-1de692d823115945c2232e1aa3066811